doc/docker/share/packelektra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+IDENTIFIER="${1}" # tag or hash of commit
+PKGNAME="elektra"
+PKGVERSION="${IDENTIFIER}"
+RELEASE="1"
+SOURCE="https://codeload.github.com/ElektraInitiative/libelektra/tar.gz/${IDENTIFIER}"
+ELEKTRADIR="libelektra-${IDENTIFIER}"
+
+# install elektra and create deb package
+cd /share
+if [ ! -d "${ELEKTRADIR}" ]; then
+    curl -sS ${SOURCE} | tar xz --transform "s/libelektra-${IDENTIFIER}/${ELEKTRADIR}/"
+fi
+if [ ! -d "{ELEKTRADIR}/build" ]; then
+  mkdir "${ELEKTRADIR}/build"
+fi
+cd "${ELEKTRADIR}/build"
+cmake -DCMAKE_INSTALL_PREFIX=/usr .. && make
+checkinstall -D -y --pkgname "${PKGNAME}" --pkgversion "${PKGVERSION}" --pkgrelease "${RELEASE}" --pkgsource "${SOURCE}" --pakdir /share
```

Comment:
Takes more time than just `make install` but I wanted to demonstrate that you can create customized packages.